packages/web-components/fast-components/src/anchor/anchor.definition.ts

```diff
@@ -38,49 +39,56 @@ export const fastAnchorDefinition: WebComponentDefinition = {
                 },
                 {
                     name: "download",
-                    description: "The download attribute",
+                    title: "Download",
+                    description: "The HTML download attribute of the anchor",
                     type: DataType.string,
                     default: undefined,
                     required: false,
                 },
                 {
                     name: "href",
-                    description: "The href attribute",
+                    title: "Href",
```

Comment:
I think we can make a more friendly name for this
```suggestion
                    title: "URL",
```